pkg/policy/validate.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/distribution/distribution/reference"
 	jsonpatch "github.com/evanphx/json-patch/v5"
+	"github.com/gardener/controller-manager-library/pkg/logger"
```

Comment:
wrong logger lib got imported ?

Reply:
Merging this to cherry-pick the changes https://github.com/kyverno/kyverno/pull/5311 of PR